src/reason_parser.mly

```diff
@@ -1617,8 +1617,8 @@ nonlocal_module_binding_details:
 non_arrowed_module_type: mark_position_mty(_non_arrowed_module_type) {$1}
 _non_arrowed_module_type:
   | simple_module_type {$1}
-  | MODULE TYPE module_expr %prec below_LBRACKETAT
-      { mkmty(Pmty_typeof $3) }
+  | MODULE TYPE OF module_expr %prec below_LBRACKETAT
```

Comment:
Should we use 
```
| MODULE TYPE OF? module_expr..
```
to make this optional? 
Is there any code out there is written as "include module type Sting" ?